src/components/Books.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable react/jsx-key */
```

Comment:
- [ ] Please refrain from using any `eslint-disable` in your code, as it defies the purpose of using linters in the first place.